### DISCUSSION TITLE
[Projects Beta] Closing a milestone doesn't remove issue from a project

### DISCUSSION THREAD

**Discussion by pavlenex** [community/community#7535] [Projects and Issues]
We use `milestones` to distinguish different releases. When a release we close a milestone, when a milestone is closed, issues still remain in the project. 

I think the least we can do is add a collapsible view for closed milestone.

![Screenshot 2021-11-10 at 11 35 14](https://user-images.githubusercontent.com/36959754/141097815-69035934-36dd-47db-8d95-d66cb99e66ce.png)

**Comment by azenMatt** [comment]
Hi @pavlenex 

We aren't currently planning on removing things from the project based on closing the milestone, but a couple of options for you: 

- You could archive all the items from the project board that are in the milestone 
- You could filter the milestone out off the view with the `-` filter

**Reply by danielnordh** [reply]
Not removing issues or hiding milestones automatically when completed will get out of hand REAL quick.
It was the first thing that jumped out at me when moving over to the Beta. Please reconsider this, doing manual cleanup is not sustainable.

My suggestion would be to not remove issues from the project, but hide completed milestones automatically in the dashboard.

**Reply by nvie** [reply]
Agreed, being able to add a filter to hide/exclude closed milestones would be amazing, rather than having to filter them manually with `-milestone:1.0 -milestone:1.1 -milestone:1.2 -milestone:2.0` etc. (Which _does_ work as a workaround, though.)
